types/highland/index.d.ts

```diff
@@ -884,7 +948,82 @@ declare namespace Highland {
 		 *
 		 * _([1, 2, 3, 4]).scan1(add) // => 1, 3, 6, 10
 		 */
-		scan1<U>(memo: U, x: (memo: U, x: R) => U): Stream<U>;
+		scan1<U>(x: (memo: U, x: R) => U): Stream<U>;
```

Comment:
likewise with the name iterator

Reply:
Similar situation to reduce1 -- how about

```
scan1<U>(iterator: (a: R | U, x: R) => U): Stream<U>;
```

And as tests are already missing lets add these:

```

// $ExpectType Stream<Foo>
fooStream = fooStream.scan1((memo, x) => {
  return memo;
});

// $ExpectType Stream<Bar>
barStream = fooStream.scan1((memo, x) => {
  return bar;
});
```